code/__DEFINES/supermatter.dm

```diff
@@ -124,10 +124,6 @@
 
 /// The divisor scaling value for cubic power loss.
 #define POWERLOSS_CUBIC_DIVISOR 500
-/// The power threshold required to transform power loss into a linear function. It is the power needed for the derivative of the cubic power loss to be equal to POWERLOSS_LINEAR_RATE.
-#define POWERLOSS_LINEAR_THRESHOLD 5880.76
-/// The offset for the linear power loss function. It is the power loss when power is at POWERLOSS_LINEAR_THRESHOLD.
-#define POWERLOSS_LINEAR_OFFSET 1627.01
 /// The rate at which the linear power loss function scales with power.
```

Comment:
out of interest, why turn these into variables, couldn't you embed the math operations inside them, so you don't need to change them, but they're still treated as constants?